DeepSpeech.py

```diff
@@ -1121,6 +1129,7 @@ def start_coordination(self, data_sets, step=0):
             # We only have to train, if we are told so and are not at the target epoch yet
             self._train = FLAGS.train and self._target_epoch > self._epoch
             self._test = FLAGS.test
+            self._dev = FLAGS.dev
```

Comment:
I intended to define it from User parameter, but I guess it will be better to enable self._dev at the time when I am adding task for Validation part in an epoch_running

"if FLAGS.validation_step > 0 and (FLAGS.validation_step == 1 or self._epoch > 0) and self._epoch % FLAGS.validation_step == 0:
                     # The current epoch should also have a validation part
                     self._epochs_running.append(Epoch(self._epoch, self._num_jobs_dev, set_name='dev', report=is_display_step))
                    # Something like this and then disabling it after I am done with validation part and enabling the self._train again.
                     self._dev = True
                     self._train = False 
"

Would that be a good idea because it would mean playing and toggling with the user defined parameter self._train at the same time for moving in between state machines.